### PR TITLE
bugfix/travis fix with updated buildtools

### PR DESCRIPTION
**Pull request by kvithayathil**
- Updated jacoco-android plugin to address failing build due to [a known issue](https://github.com/arturdm/jacoco-android-gradle-plugin/issues/42)
- Version bumps on contraintLayout and Room database

### CODE REVIEW

File: app/src/main/java/com/koalatea/thehollidayinn/softwareengineeringdaily/notifications/DailyAlarmReceiver.kt [hunk 1]
```diff
@@ -55,8 +55,7 @@ class DailyAlarmReceiver: BroadcastReceiver() {
         val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            val importance = NotificationManagerCompat.IMPORTANCE_DEFAULT
-            val channel = NotificationChannel(CHANNEL_ID, CHANNEL_ID, importance)
+            val channel = NotificationChannel(CHANNEL_ID, CHANNEL_ID, NotificationManager.IMPORTANCE_DEFAULT)
```
Comment (TheHollidayInn) [inline app/src/main/java/com/koalatea/thehollidayinn/softwareengineeringdaily/notifications/DailyAlarmReceiver.kt]: Can we keep these separated?

File: app/schemas/com.koalatea.thehollidayinn.softwareengineeringdaily.data.AppDatabase/1.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (TheHollidayInn) [inline app/schemas/com.koalatea.thehollidayinn.softwareengineeringdaily.data.AppDatabase/1.json:1]: Is this file required?

Comment (kvithayathil) [reply app/schemas/com.koalatea.thehollidayinn.softwareengineeringdaily.data.AppDatabase/1.json:1]: It's auto-generated by room. It can help with schema migrations. It can be turned off, but it's probably not recommended. See this thread - https://stackoverflow.com/q/44322178
